lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1753,6 +1753,22 @@ defmodule Kernel.SpecialForms do
   catch values also support `:error`, as in Erlang, although it is
   commonly avoided in favor of raise/rescue control mechanisms.
 
+  ## After clauses
+
+  An `after` clause works like an `ensure` or `finally` clause in other languages.
```

Comment:
I'm 👎 on mentioning other languages, I think people can arrive to this conclusion pretty easily and we don't mention the connection with `try` and similar constructs in other languages. Wdyt?